src/test/java/StepDefinition/Darksky/LoginSD.java

```diff
@@ -1,23 +1,65 @@
 package StepDefinition.Darksky;
 
+import Helper.Misc;
+import Pages.Darksky.LandingPageHW5;
 import Web.MyDriver;
+import io.cucumber.java.en.And;
 import io.cucumber.java.en.Given;
 import io.cucumber.java.en.Then;
 import io.cucumber.java.en.When;
+import org.testng.Assert;
 
-public class LoginSD {
-    @Given("I am on Darksky")
+public class LoginSD<tab> {
+    LandingPageHW5 lpage = new LandingPageHW5();
+
+    @Given("I am on Darksky.com")
     public void launchDarksky(){
-        MyDriver.launchUrlOnNewWindow("https://www.darksky.net/");
+//        MyDriver.launchUrlOnNewWindow("https://www.darksky.net/");
+        lpage.launchDarksky();
+    }
+    @When("^I scroll down to TimeMachine Button$")
+    public void scrollToTM(){
+//        lpage.scrollToTodayBarByPixel();
+       lpage.scrollToTimeMachineBtnByPixel();
+        Misc.pause(3);
```

Comment:
If you use FluentWait driver you should not need more wait to add but if still required pls use explicit wait, its dynamic and more efficient.